session/bootstrap.go

```diff
@@ -1989,6 +1994,39 @@ func upgradeToVer98(s Session, ver int64) {
 	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN IF NOT EXISTS `Token_issuer` varchar(255)")
 }
 
+func upgradeToVer99Before(s Session, ver int64) bool {
+	if ver >= version99 {
```

Comment:
If lager than version99,  seems it should be set tidb_enable_metadata_lock for upgrade?